news/_posts/releases/2022-08-08-prestashop-8-0-beta-release.md

```diff
@@ -0,0 +1,111 @@
+
+The main new features for this version include an update to [Symfony 4.4](https://symfony.com/releases/4.4), compatibility with [PHP 8 and 8.1](https://www.php.net/), an enhanced experimental product page, and a new security page which allows shop administrators to manage user sessions and configure their password policy.
+
+In addition, this version fixes [over 130 bugs](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+is%3Aclosed+milestone%3A8.0.0+label%3ABug+-label%3ARegression+-label%3A8.0.x+-label%3Adevelop) found in previous releases, including:
+
+* [In some circumstances, customers could be able to accidentally order an out-of-stock product, even if it was not allowed](https://github.com/PrestaShop/PrestaShop/issues/10762).
+* [An empty cart was being created whenever a customer signed in](https://github.com/PrestaShop/PrestaShop/issues/9589).
+* [Canonical links were missing on several pages](https://github.com/PrestaShop/PrestaShop/issues/9503).
+* [An error message was being logged every time an order was paid, even though there was no error](https://github.com/PrestaShop/PrestaShop/issues/9866).
+* [Visitors having placed an order weren't able to set up a customer account using the form at the bottom of the order confirmation page](https://github.com/PrestaShop/PrestaShop/issues/14414).
+
+PrestaShop 8 is also the project's first company-neutral release, meaning that it no longer includes commercial products and services built-in. You can find out more by reading [Beyond 1.7](/news/prestashop-beyond-1-7/) and watching the project's [public demo from November 2021](https://www.youtube.com/watch?t=128&v=jwOy3gx-MNs&feature=youtu.be).
+
+This is the first major version of PrestaShop since 1.7.0 was released back in 2016. A significant cleanup has been performed in the codebase, which means several internal components have been slightly modified, and many older, deprecated components have been removed. This article in DevDocs describes the full list of what has been changed: [Changes in 8.0](https://devdocs.prestashop.com/8/modules/core-updates/8.0/).
+
+**Remember, this beta version is pre-release software and it is highly possible that you will encounter some issues. Do not use it in your production shop!**
+
+Also note that it is not possible to upgrade a shop from a beta to the final version using the upgrade module.
+
+## Your feedback is important!
+
+The public beta period is an extremely important phase to prepare for the upcoming release. During this period, we encourage everyone in the community to download this version and test how it behaves in their unique environment.
+
+Here are some examples of how you can help:
+
+* If you have access to a real shop, set up a copy of it in a separate environment and update it to PrestaShop 8 using the [upgrade module](https://github.com/PrestaShop/autoupgrade/releases). See if you notice any issues on the shop compared to your previous version.
```

Comment:
ps_metrics, ps_facebook, Prestashop Checkout, Google marketing, welcome, mbo

Module list to uninstall before upgrading